tasks/requirements_debian.yml

```diff
@@ -16,6 +16,7 @@
       - gcc
       - gettext
       - git
+      - npm
```

Comment:
Included this in a separate commit, just to get it merged while we talk about the rest here.